.travis.yml

```diff
@@ -3,17 +3,19 @@ language: php
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
     - php: 7.2
+    - php: 7.3
+    - php: 7.4snapshot
```

Comment:
@szepeviktor Is it possible to simply test against the `7.4` version instead of this "snapshot" ? Also, we can remove the `7.2` version as this is no longer supported on upcoming release.

Reply:
I think that is waiting to be released https://docs.travis-ci.com/user/reference/bionic/#php-support

Reply:
Hmm I thought it was now released as described in this post: https://travis-ci.community/t/php-7-4-release/6197 Perhaps their documentation is not yet up to date?! I dunno. But looking at other projects requiring php, `7.4` seems to work.

Reply:
Done.